Faraday_Proxy_Tools/FaradayIO/commandmodule.py

```diff
@@ -131,6 +131,55 @@ def create_fixed_length_packet(data, fixed_legth):
         return padded_data
 
 
+def create_fixed_length_packet_padding(data, fixed_legth, padding_byte):
```

Comment:
I see your point for Zfill on the leading zeros but what about trailing? I can use Zfill and remove trailing zero support completely but how would I ensure the size 4 bytes on righthand decimal side?